cmd/snapweb/handlers.go

```diff
@@ -122,6 +127,37 @@ func initURLHandlers(log *log.Logger) {
 	http.HandleFunc("/", makeMainPageHandler())
 }
 
+func createUserHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This function feels unnecessarily complex, especially given that there will be more api endpoints that will be imported. I would suggest to try a "passthrough" thing that just takes whatever is on the http socket and passes it on to the unix socket. I outlined what I have in mind in https://github.com/david-barth-canonical/snapweb/pull/1. It may need some additional work but it should give you a good idea. It also needs tests and testing.